core/templates/dev/head/pages/topic_editor/main_editor/save_pending_changes_modal_directive.html

```diff
@@ -1,6 +1,6 @@
 <div class="modal-body">
   <p>
-    Please save all pending changes before creating a story.
+    Please save all pending changes before creating or opening a story.
```

Comment:
What does "opening" a story mean?

Why do changes need to be saved before opening a story? (I understand it for creating but I'm not sure I understand it for opening.)

Reply:
Changed the displayed message.

Reply:
But nothing is going to stop the user opening the story editor in a new tab, right?

What if the user manually enters the story editor URL in a new tab? Do we auto-redirect them to the topic editor in that case?

Reply:
No, if it's in a separate tab, it's fine, since the topic editor would still be shown. The problem was that since topic editor was no longer present when story editor was opened in the same tab, so all pending changes were gone.
Or, how about we open story editor in new tab regardless (even when the user clicks on Create Story)? Then, I don't think this modal would be required.

Reply:
I don't know... let's ask the team? Maybe you can give a quick demo in the meeting and ask for thoughts.

I'm happy with merging this in the meantime and doing follow-up changes later, if you like.